json_journal/tabulate_data.py

```diff
@@ -1,6 +1,23 @@
+'''Tabulate data'''
 from tabulate import tabulate
 
+
 def tabulate_sorted_entries(all_entries, sorted_accounts):
-        output_rows = [[account_name, all_entries[account_name]] for account_name in sorted_accounts]
-        output_headers = ["Account", "Amount"]
-        print(tabulate(output_rows, headers=output_headers))
\ No newline at end of file
+    '''
+     Show balances in a table sorted alphabetically by account name.
+
+    Parameters
+    ----------
+    All_entries : dict
+        Contains the full account name and corresponding balance.
+    Sorted_accounts : list
+        Contains the parent account names sorted alphabetically.
+
+    Returns
```

Comment:
Strictly speaking, this function doesn't return anything. We have three options:
1. Return the table so that it can be printed in main,
2. Change this documentation to better explain that the function won't return, but will print to the terminal,
3. Anything you can think of.

Reply:
Done